crates/api_models/src/user.rs

```diff
@@ -306,7 +306,7 @@ pub struct OpenIdConnectPublicConfig {
     pub name: OpenIdProvider,
 }
 
-#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, strum::Display)]
 #[serde(rename_all = "snake_case")]
```

Comment:
Can you add `#[strum(serialize_all = "snake_case")]`. So that you don't have to do lower case and it will be same as serde.